golden_marshmallows/schema.py

```diff
@@ -5,7 +5,9 @@
 from sqlalchemy.dialects.postgresql import (
     ARRAY as pgARRAY, BIGINT, ENUM, TIMESTAMP, UUID)
 from sqlalchemy.sql.sqltypes import (
-    ARRAY, Boolean, BOOLEAN, DATE, Integer, INTEGER, JSON, String, TEXT)
+    ARRAY, Boolean, BOOLEAN, DATE, DateTime, Integer, INTEGER, JSON, String, TEXT, Text)
+from sqlacodegen.dialects.postgresql import LTREE
+from sqlalchemy_utils import LtreeType
```

Comment:
For these two imports, since they are not part of the base requirements for this project, could you make them optional like so?
```python
try:
    from sqlacodegen.dialects.postgresql import LTREE
    _SQLACODEGEN = True
except ImportError:
    _SQLACODEGEN = False

try:
    from sqlalchemy_utils import LtreeType
    _SQLALCHEMY_UTILS = True
except ImportError:
    _SQLALCHEMY_UTILS = False
```
Then further down, _after_ `FIELD_TYPE_MAP` is defined, you can do:
```python
if _SQLACODEGEN:
    FIELD_TYPE_MAP[LTREE] = fields.String

if _SQLALCHEMY_UTILS:
    FIELD_TYPE_MAP[LtreeType] = fields.String
```